### PR TITLE
docs: fix repo name from kbs to Trustee

### PR DESCRIPTION
**Pull request by Xynnn007**
This commit is for the name-changing of kbs repo.

Related to #250

### CODE REVIEW

**Comment by Xynnn007** [comment]
Well. The link check must fail as we haven't change the repo name for now.

**Review by mkulke — COMMENTED** [review]
i suggest we drop the capitalization and technical labels (url, crates identifier, directories). doesn't harm probably, but it's odd

**Comment by Xynnn007** [comment]
> i suggest we drop the capitalization and technical labels (url, crates identifier, directories). doesn't harm probably, but it's odd

I got your point about capitalization. What does "technical labels" mean? Could you help to give a suggested fix example?

**Comment by mkulke** [comment]
> > i suggest we drop the capitalization and technical labels (url, crates identifier, directories). doesn't harm probably, but it's odd
> 
> I got your point about capitalization. What does "technical labels" mean? Could you help to give a suggested fix example?

ah, by "technical label" I mean everything but text (e.g. in markdown), but urls, crates, folders should all be `trustee` IMO, not `Trustee`.

**Comment by Xynnn007** [comment]
@mkulke Updated.

**Review by mkulke — APPROVED** [review]
lgtm

File: attestation-service/docs/grpc-as.md [hunk 1]
```diff
@@ -14,16 +14,16 @@ Here are the steps of building and running gRPC Attestation Service:
...
 WORKDIR=$(pwd)
 make && make install
 ```
 
 Build and run container image
 ```shell
-git clone https://github.com/confidential-containers/kbs
-cd kbs
+git clone https://github.com/confidential-containers/trustee
+cd trustee/attestation-service
```
Comment (fitzthum) [inline attestation-service/docs/grpc-as.md:26]: should this be `cd trustee`? In the next line we specify `attestation-service` as part of the path to the Dockerfile

File: attestation-service/docs/restful-as.md [hunk 2]
```diff
@@ -10,16 +10,16 @@ Here are the steps of building and running RESTful Attestation Service:
...
 WORKDIR=$(pwd)
 make && make install
 ```
 
 Build and run container image
 ```shell
-git clone https://github.com/confidential-containers/kbs
-cd kbs
+git clone https://github.com/confidential-containers/trustee
+cd trustee/attestation-service
```
Comment (fitzthum) [inline attestation-service/docs/restful-as.md:22]: Same as above

**Review by fitzthum — APPROVED** [review]
One comment but looks good in general

File: attestation-service/docs/grpc-as.md (hunk 1 above)
Comment (Xynnn007) [reply attestation-service/docs/grpc-as.md:26]: Oh yes. Thanks for pointing out

**Comment by Xynnn007** [comment]
Hi folks, I've fixed due to suggestions. Maybe we can merge this together with the name changing?

**Pull request merged**